qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -1469,6 +1469,10 @@ def run_js_async(self, code, callback=None, *, world=None):
             self._widget.page().runJavaScript(code, world_id, callback)
 
     def reload(self, *, force=False):
+        original = urlutils.extract_error_url(self.raw_url())
+        if original:
+            self.load_url(original[0])
+            return
```

Comment:
Why only for QtWebEngine?

Reply:
Perhaps because the original issue is tagged with QtWebEngine?...

On webkit, it seems that there isn't any problem  (but I'm not entirely sure about this. JavaScript must be enabled for the try again button to work.)

Reply:
Okay, so either this should be implemented for QtWebEngine only, or both, but not a mixture of those things like now (with `BrowserTab.url()`.

I'd prefer having the same implementation in both cases, and FWIW opening `qute://error?url=https://example.org&error=test` works as expected for me.